app/src/main/res/values/strings.xml

```diff
@@ -14,4 +15,7 @@
     <string name="close">Close</string>
     <string name="save_to_library">Save to Library</string>
     <string name="share">Share</string>
+    <string name="loading">Loading</string>
+    <string name="stop">STOP</string>
```

Comment:
để string k viết hoa
viết hoa toàn bộ ở trong text view bằng cách android:textAllCaps="true"